resources/ext/build-scripts/install-dropsonde.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+FORGE_API="forgeapi-cdn.puppet.com"
+MODULE_SLUG="puppetlabs-dropsonde"
+MODULE_NAME="dropsonde"
+MODULE_VERSION="0.0.5"
+
+curl -O "https://${FORGE_API}/v3/files/${MODULE_SLUG}-${MODULE_VERSION}.tar.gz"
+mkdir "${DESTDIR}/opt/puppetlabs/server/data/puppetserver/puppetserver_modules"
```

Comment:
Where does `$DESTDIR` come from?

Reply:
also might be good to put `${DESTDIR}/opt/puppetlabs/server/data/puppetserver/puppetserver_modules` into a variable for re-use.

Reply:
My inspiration was from https://github.com/puppetlabs/puppetserver/blob/5f51e1a301312327b18bb2ffa565b1642ff7c4c6/resources/ext/build-scripts/install-vendored-gems.sh#L17 the script used to install vendored gems.
`DESTDIR` i think is an environment variable.
That is a good idea to use a variable for re-use.